diff --git a/weed/storage/erasure_coding/ec_encoder.go b/weed/storage/erasure_coding/ec_encoder.go
index eeeb156e6..51aa8925e 100644
--- a/weed/storage/erasure_coding/ec_encoder.go
+++ b/weed/storage/erasure_coding/ec_encoder.go
@@ -15,8 +15,8 @@ import (
 )

 const (
-       ParityShardsCount           = 4
+       ParityShardsCount           = 2
        TotalShardsCount            = DataShardsCount + ParityShardsCount
        MinTotalDisks               = TotalShardsCount/ParityShardsCount + 1
        ErasureCodingLargeBlockS
